### PR TITLE
Add AnimeJunkies Presence

### PR DESCRIPTION
**Pull request by RedstoneZockt**
- Add Anime List Display
- Add article Display 
- Add Foren Display

Proof:
Beitrag
Foren
Startseite
Anime Liste

### CODE REVIEW

File: websites/A/AnimeJunkies/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+      "name": "RedstoneZockt",
+      "id": "355333222596476930"
+    },
+    "service": "AnimeJunkies",
+    "description": {
+      "en": "Official Presence for AnimeJunkies"
+    },
```
Comment (Bas950) [inline websites/A/AnimeJunkies/dist/metadata.json:10]:
```suggestion
    "description": {
      "en": "The community website about anime!",
      "de": "Die Communitywebseite über Anime!",
      "nl": "De communitywebsite over anime!"
    },
```
Official description

File: websites/A/AnimeJunkies/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "url": ["animejunkies.tv", "www.animejunkies.tv"],
+    "version": "0.0.1",
+    "logo": "https://animejunkies.tv/wp-content/uploads/2020/05/AnimeJunkies-Logo.png",
+    "thumbnail": "https://cloud.animejunkies.tv/index.php/s/PNG5APC7j5J8Y6H/preview",
+    "color": "#ff0000",
+    "tags": ["anime"],
+    "category": "anime",
+    "readLogs": false
```
Comment (Bas950) [inline websites/A/AnimeJunkies/dist/metadata.json:18]:
```suggestion
    "category": "anime"
```

File: websites/A/AnimeJunkies/presence.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+let last_path = "",
+  time_stamp: number = Date.now();
+
+presence.on("UpdateData", async () => {
+  const presence_data = {
+    details: "",
+    state: "",
+    smallImageKey: "",
+    smallImageText: ""
+  };
```
Comment (Bas950) [inline websites/A/AnimeJunkies/presence.ts:12]: dont set it to "", just dont set it at all at start...

File: websites/A/AnimeJunkies/presence.ts [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  }
+
+  const presenceData: PresenceData = {
+    largeImageKey: "logo",
+    smallImageKey: presence_data.smallImageKey,
+    smallImageText: presence_data.smallImageText,
+    details: presence_data.details,
+    state: presence_data.state,
+    startTimestamp: time_stamp
+  };
```
Comment (Bas950) [inline websites/A/AnimeJunkies/presence.ts:57]: just use 1 PresenceData object...

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/A/AnimeJunkies/presence.ts [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    if (
+      x.length != 0 &&
+      document.getElementsByClassName("author-box-header").length != 0
+    ) {
+      const b = x[0];
+      presenceData.details = "Schaut einen Beitrag an";
+      presenceData.state = b.textContent;
+    } else {
+      presenceData.details = "Erkundet AnimeJunkies";
+      presenceData.state = "";
```
Comment (Bas950) [inline websites/A/AnimeJunkies/presence.ts:37]: Dont set empty state

**Review by Bas950 — CHANGES_REQUESTED** [review]
Add some more things to the presence...

**Comment by RedstoneZockt** [comment]
The main of the website are news posts (over 400), They where automaticly displayed by the title.
There are other thinks to display in the future, but the sub domain website is not finished yet and is in closed beta. so i can't add them now to the presence.

If it finished, il add more thinks.

**Comment by ririxidev** [comment]
For now; this presence is not qualified since it doesn't do a lot. Sorry.

**Comment by RedstoneZockt** [comment]
Okay I will work on it.

**Pull request closed without merging**